layouts/partials/comments/provider/cactus.html

```diff
@@ -0,0 +1,27 @@
+{{- with .Site.Params.comments.cactus -}}
+<style>
+    .cactus-editor-textarea {
+        color: var(--body-text-color);
+    }
+
+    .cactus-comment-header {
+        color: var(--card-text-color-main);
+    }
+
+    .cactus-message-text>p {
+        color: var(--body-text-color);
+    }
+</style>
+
```

Comment:
You need to load external dependencies using `{{- partial "helper/external" (dict "Context" . "Namespace" "Cactus") -}}`